examples/configuration.nix

```diff
@@ -37,25 +37,32 @@
   # Enable this module to use clightning, a Lightning Network implementation
   # in C.
   services.clightning.enable = true;
-  # == TOR
-  # Enable this option to announce our Tor Hidden Service. By default clightning
-  # offers outgoing functionality, but doesn't announce the Tor Hidden Service
-  # under which peers can reach us.
-  # services.clightning.announce-tor = true;
+  #
+  # Set this to create an onion service by which clightning can accept incoming
+  # connections via Tor.
+  # nix-bitcoin.onionServices.clightning.enable = true;
+  #
+  # Set this to enable clightning to announce the onion service to other peers.
+  # nix-bitcoin.onionServices.clightning.public = true;
```

Comment:
Somebody might want to enable the onion service without announcing it though...

Reply:
Sure I'm only saying announce implies public, not the other way around. We wouldn't need to mention public in the example config, because public without announce is a rather advanced use case.

Reply:
You're right. It shouldn't be possible to set `onionServices.service.public` when `onionServices.service.enable = false` anyway.

Reply:
Also we need to change the wording in `README.md` to reflect that onionServices are no longer created for every service automatically. Ex:

`All applications use Tor for outbound connections and accept inbound connections via onion services.` -> `All applications use Tor for outbound connections and support accepting inbound connections via onion services.`

Reply:
Fixed in 6495bf5aaa19c856f8514d1bdba335a233f7c5eb.